frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -67,6 +57,6 @@ export const OCSStorageClassDropdown: React.FC<OCSStorageClassDropdownProps> = (
 };
 
 type OCSStorageClassDropdownProps = {
-  onChange: React.Dispatch<React.SetStateAction<string>>;
+  onChange: (name: string, provisioner: string) => void;
```

Comment:
This will cause problems with the existing code.

Reply:
same as above

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx#L329